src/main/java/racingcar/RacingRule.java

```diff
@@ -0,0 +1,27 @@
+package racingcar;
+
+import java.util.List;
+import racingcar.domain.Car;
+
+public class RacingRule {
+
+  public void play(int trynum, List<Car> cars) {
+    for (int i = 0; i < trynum; i++) {
+      System.out.println("실행 결과");
+      ResultView resultView = new ResultView();
+      resultView.printResult(cycleCarMove(cars));
+    }
+  }
+
+  private static List<Car> cycleCarMove(List<Car> cars) {
+
+    for (Car car : cars) {
+      RandomMovingStrategy randomMovingStrategy = new RandomMovingStrategy();
+      randomMovingStrategy.move(car, randomMovingStrategy.getRandNum(),
+          car.getcriteriaNum());
```

Comment:
```suggestion
      randomMovingStrategy.move(car, randomMovingStrategy.getRandNum(), car.getcriteriaNum());
```
진행하시면서 출력할 때 외엔 getter, setter 사용을 지양해주세요 :)